crashtracker/src/crash_info.rs

```diff
@@ -234,4 +272,13 @@ impl CrashInfo {
             Ok(Some(self.upload_to_dd(endpoint, timeout)?))
         }
     }
+
+    pub fn upload_to_telemetry(&self, config: &CrashtrackerConfiguration) -> anyhow::Result<()> {
+        if let Some(metadata) = &self.metadata {
+            if let Ok(uploader) = TelemetryCrashUploader::new(metadata, config) {
+                uploader.upload_to_telemetry(self, config.timeout)?;
+            }
+        }
```

Comment:
Do we want to do nothing if there is no crash tracking metadata ?

Reply:
I'm not sure how useful a crashreport with no metadata would be.  If you think it would be helpful, we can make this an optional field